FSharpBuild.Directory.Build.props

```diff
@@ -15,7 +15,9 @@
     <ProtoOutputPath>$(ArtifactsDir)\Bootstrap</ProtoOutputPath>
     <ValueTupleImplicitPackageVersion>4.4.0</ValueTupleImplicitPackageVersion>
     <WarningsAsErrors>1182;0025;$(WarningsAsErrors)</WarningsAsErrors>
-  </PropertyGroup>
+    <OtherFlags>$(OtherFlags) --nowarn:3384</OtherFlags>
+    <NoWarn>$(NoWarn);FS3384</NoWarn>
```

Comment:
I'm concerned about the duplication.

Reply:
well, I forgot to delete line 18, cos I was sleepy, will take care of it.